packages/ui/src/Toolbar/index.js

```diff
@@ -127,9 +133,9 @@ class Raw extends Component {
             )
           })}
         </List>}
-        {layout.length && <List subheader={
-            <ListSubheader>Layout plugins</ListSubheader>
-          }>
+        {layout.length > 0 && <List subheader={
```

Comment:
`content.length > 0` is the same as `boolean(content.length)`, no?

Reply:
yeah should be, I just prefer this syntax

Reply:
Let's keep it how it was :)